engine/src/main/java/org/entando/entando/web/page/PageController.java

```diff
@@ -138,12 +175,17 @@ public void setPageValidator(PageValidator pageValidator) {
             throw new ValidationGenericException(bindingResult);
         }
         this.getPageService().removePage(pageCode);
-        return new ResponseEntity<>(new RestResponse(pageCode), HttpStatus.OK);
+        DeleteDto dto = new DeleteDto(pageCode);

```

Comment:
see comment above. Refactor this class out and either return a sparse Page or return a Map